ngraph/core/src/itt.hpp

```diff
@@ -31,7 +33,33 @@ namespace ngraph
         {
             OV_ITT_DOMAIN(nGraph);
             OV_ITT_DOMAIN(nGraphPass_LT);
-            OV_ITT_DOMAIN(nGraphOp, "nGraph::Op");
+            OV_ITT_DOMAIN(ngraph_op, "nGraph::Op");
         }
     }
+    OV_CC_DOMAINS(ngraph_op);
 }
+
+#if defined(SELECTIVE_BUILD) || defined(SELECTIVE_BUILD_ANALYZER)
+#define NGRAPH_OP_SCOPE(region, ...) OV_SCOPE(ngraph_op, region, __VA_ARGS__)
+#else
+#define NGRAPH_OP_SCOPE(region, ...)                                                               \
```

Comment:
Would it be possible to change this macro in a way that doesn't use VA_ARGS? The current form makes the code a bit confusing at the call site, for example here https://github.com/openvinotoolkit/openvino/blob/21850b5f6dfd8969d345fc3661a9328bab66bf09/ngraph/core/src/op/broadcast.cpp#L222-L224 

I would really like to see it in use like this:

```
NGRAPH_OP_SCOPE(v3_Broadcast_evaluate) {
    (...)
    return op::util::BroadcastBase::evaluate_broadcast(
        inputs[0], outputs[0], pair_broadcast_axes, result_shape.to_shape());
}
```

Reply:
@tomdol Yes I also found this code. It happened after the clang-format. I will try to disable clang-format inside these macros.
In general I need to control start and end points, it is why I put all code inside the macro.

Reply:
I cannot automatically disable clang-format from macro. I will create functions in order to have a good code style.

Reply:
Fixed